sdk/iot/core/doc/mqtt_state_machine.md

```diff
@@ -136,3 +138,120 @@ _Example:_
         }
     }
 ```
+
+### Retrying Operations
+
+The IoT Core SDK documents and provides APIs for the error policy and timing policies separately. We are also supplying optional APIs for error classification and retry timing.

```

Comment:
The IoT device library also supply's optional APIs for error classification and retry timing.

Reply:
Agreed - I will remove the "provides APIs" from the first sentence and replace the second with your proposal.

Reply:
Reformulated.

Reply:
I'm not sure what IoT Core SDK refers to. If this is my team's core then this is "Azure C SDK's Core library" as it is not IoT-specific. If this is not my team's core, then this is not an SDK at all; but is instead a Client Library.